lib/Search/PageContentProvider.php

```diff
@@ -95,13 +95,7 @@ public function getOrder(string $route, array $routeParameters): int {
 	 * @throws FileSearchException
 	 */
 	public function search(IUser $user, ISearchQuery $query): SearchResult {
-		// Only search for page content if the app is active
-		if ($this->appManager->isEnabledForUser('circles', $user) &&
-			strpos($query->getRoute(), 'collectives.') === 0) {
-			$collectiveInfos = $this->collectiveHelper->getCollectivesForUser($user->getUID(), false, false);
-		} else {
-			$collectiveInfos = [];
-		}
+		$collectiveInfos = $this->collectiveHelper->getCollectivesForUser($user->getUID(), false, false);
```

Comment:
Do we still need the circles check or is that gracefully handled in getCollectivesForUser if circles is disabled?

Reply:
Nope, good catch. The check for circles is still required.